README.md

```diff
@@ -1,36 +1,70 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15555388&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
 
+software engineering is the systematic application of engineering principles, methods and tools to the development and maintenance of high-quality software principles.
+software engineering helps in the creating of software applications and systems that power various aspects of modern life such as transport and logistics, healthcare, banking and finance and etc.
+
 
 Identify and describe at least three key milestones in the evolution of software engineering.
 
+three key milestones in the evolution of software engineering are: 
+(i) the development of programming languages eg Fortan,C
+(ii) the establishemetn of software engineering as a discpline in the 1960s
+(iii) the rise of agile methodologies in the 2000s
 
```

Comment:
- Software Developer: Responsible for writing code and implementing software solutions.
  - Quality Assurance Engineer: Ensures software quality by designing and executing test plans.
  - Project Manager: Oversees the planning, execution, and delivery of software projects.

Reply:
The Software Developer focuses on creating and maintaining software
The Quality Assurance Engineer ensures the software meets quality standards through rigorous testing
The Project Manager oversees the project’s execution, ensuring alignment with business objectives and effective team collaboration.

Reply:
The software developer creates the software by writing the codes, the QA engineer makes sure eveerything is being run properly, and the project manager ensures that the project is running smoothly.